src/Client/ConcordiumClientOptions.cs

```diff
@@ -0,0 +1,36 @@
+
+using Grpc.Net.Client;
+
+#pragma warning disable CS8618
+
+namespace Concordium.Sdk.Client;
+
+/// <summary>
+/// Configurations to Concordium Client.
+/// </summary>
+public sealed class ConcordiumClientOptions
```

Comment:
This needs to be a class because of it usage.

The library, and hence client, will most likely be used in some form of (web) application where configurations are loaded from some file. For this to work there needs to be a parameterless constructor.
I have made test for this class which should capture the usages of it.